.github/actions/deploy-happy-stack/action.yml

```diff
@@ -64,6 +64,9 @@ inputs:
   github-repo-branch:
     description: "Branch of Github repo to deploy"
     required: false
+  slice:
```

Comment:
Can we also add space for `--slice-default-tag`? We might need this if I recall to change the default tags used by the other services when using a slice option. I'm a little fuzzy on the details, but wouldn't hurt to add

Reply:
added!